.travis.yml

```diff
@@ -29,6 +29,15 @@ matrix:
     - os: osx
       osx_image: xcode9.2
       sudo: required
+      env: SWIFT_SNAPSHOT=4.0.3
+    - os: linux
+      dist: trusty
+      sudo: required
+      env: SWIFT_SNAPSHOT=swift-4.1-DEVELOPMENT-SNAPSHOT-2018-02-06-a
```

Comment:
Can we test against the latest snapshot please?

Reply:
Updated to 03-04-a